pkg/apis/core/validation/validation_test.go

```diff
@@ -11738,6 +11738,7 @@ func TestValidateSecurityContext(t *testing.T) {
 	negativeRunAsUser := fullValidSC()
 	negativeUser := int64(-1)
 	negativeRunAsUser.RunAsUser = &negativeUser
+	isInclusiveErrorMssg := `must be between`
```

Comment:
Could I ask you to completely inline this string in the `errorDetail`?  No one else use it, so I don't see why we need to have it as a variable, it looks like premature generalization. Also another test case has an expected message as just a string.

Another tiny improvement: use double quotes (\``must be between`\` -> "must be between"). There is no special characters in the string that requires using of backticks and also it's better to be consistent with another test case.

Thank you for patience!

Reply:
no worries, fixed